pf binning PR

mverzett opened the pull request:
it includes:
   * updates in the PF binning. I now also compute global bin variables (sumpt, # of objects etc)
   * scripts and changes to run the root to numpy conversion in batch

Commit:
second version of PF binning, includes the training model global bin variables

Commit:
run root to numpy conversion in batch

jkiesele commented:
Can you please make sure there are no conflicts.
If I am responsible for it, please let me know ;)

Commit:
extend ignore file

Commit:
move training model to DeepJet_models.py

Commit:
remove function not existing any longer

Commit:
batch submission improvements

Commit:
fix merging conflicts

mverzett commented:
@jkiesele I updated the content to my latest and greatest and fixed the conflicts as well. Can you just make sure the last commit (merge conflicts fixing) does not screw anything up?
